package.json

```
@@ -30,6 +30,7 @@
         "@flowforge/nr-audit-logger": "^0.6.0",
         "@flowforge/nr-auth": "^0.6.0",
         "@flowforge/nr-storage": "^0.6.0",
+        "@flowforge/nr-theme": "^0.0.1",
```

```suggestion
        "@flowforge/nr-theme": "^0.1.0",
```

We're going to start at 0.1.0